### PR TITLE
MM-23362 MM-T1667 Add e2e test for draft with only 2 byte characters

### PR DESCRIPTION
**Pull request by josephk96**
Summary

- Created an e2e test that posts a two byte character in Korean in one channel, switch to another channel and assert that the draft icon does not exist next to the channel the test user posted in.

Ticket Link
Jira: https://mattermost.atlassian.net/browse/MM-23362
Test case: https://automation-test-cases.vercel.app/test/MM-T1667

### CODE REVIEW

File: e2e/cypress/integration/messaging/draft_with_only_2_byte_characters_spec.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                lastPostId = postId;
+            });
+        });
+    });
+
+    it('M23362 - Draft with only 2 byte characters shouldn\'t remain after posting', () => {
+        testChannel.name = testChannel.name.replace(/\s+/g, '-').toLowerCase();
+
+        // * Check that the Korean message got posted in the channel
+        cy.get(`#post_${lastPostId}`).should('exist');
```
Comment (saturninoabril) [inline e2e/cypress/integration/messaging/draft_with_only_2_byte_characters_spec.js:42]: Seems a bit short in checking whether the message got posted. I suggests to add:
```javascript
cy.get(`#postMessageText_${postId}`).should('contain', '휴');
```

**Comment by josephk96** [comment]
Thanks for the comments, pare! My bad for not having covered the last step :/

File: e2e/cypress/integration/messaging/draft_with_only_2_byte_characters_spec.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            cy.postMessage('휴');
+
+            // # Assign lastPostId variable to the id of the last post
+            cy.getLastPostId().then((postId) => {
+                lastPostId = postId;
+            });
+        });
+    });
+
+    it('M23362 - Draft with only 2 byte characters shouldn\'t remain after posting', () => {
```
[the remark this answers is not in the document]
Comment (saturninoabril) [reply e2e/cypress/integration/messaging/draft_with_only_2_byte_characters_spec.js:38]: Oopss, sorry I updated my comment here.

**Comment by josephk96** [comment]
I fixed the test key :)

**Comment by josephk96** [comment]
Thank you for the kind comment and taking the time to review my PR, @srkgupta  !

**Pull request merged**